scripts/shutdown.js

```diff
@@ -0,0 +1,28 @@
+// Description
+//   Shutdown uqcsbot
+//
+// Commands:
+//   `!uqcsbot shutdown` - Shuts down uqcsbot, waiting to be restarted via CESI
+
+var admin = ['trm', 'mcdee', 'csa', 'guthers', 'artemis', 'rob', 'mb'];
+
+module.exports = function (robot) {
+    robot.respond(/!?uqcsbot shutdown/i, function (res) {
+        comandee = res.message.user.name
```

Comment:
Can't we use `res.message.user.is_admin`?

Reply:
Done